src/Microsoft.VisualStudio.Services.Agent/Util/PowerShellExeUtil.cs

```diff
@@ -54,7 +54,8 @@ public string GetPath()
                     (int)install != 1)
                 {
                     Trace.Info("Not installed.");
-                    continue;
+                    // warn only.  install flag doesn't exist on onecore based editions of windows. and, there's really
+                    // no reason to check.  if subsequent keys are missing or the .exe won't run an error will be emitted at that point
```

Comment:
Should we just remove this whole install flag check if it is not serving any useful purpose?

Reply:
I can't think of a reason. (Well, the one I can think of is that no .NET = no PowerShell, which will make PS-based tasks blow up. But that's probably an acceptable risk.)

Reply:
note that the subsequent check for the powershell version will catch the case where powershell isn't installed at all.  it's just that onecore editions that include .NET Core inbox don't set that install flag for whatever reason.

Reply:
awesome! would you mind pushing a commit that removes this block entirely?

Reply:
done.